src/index.js

```diff
@@ -32,30 +36,56 @@ const icalPromises = icalUrls.map(icalUrl => {
                 sameElse: 'DD/MM'
               });
 
-              vacations.push(`*${ev.attendee.params.CN}* is on leave today (last day of leave is ${lastDayOfHoliday})`);
+              calEvents["leave"].push(`>*${ev.attendee.params.CN}* (last day of leave is ${lastDayOfHoliday})`);
+            }
+          }
+
+          if (data[k].categories.indexOf('Public Holiday') > -1 ) {
+            const start = moment(ev.start);
+
+            if (start.isSame(today, 'day')) {
+              if (!calEvents["publicHolidays"].hasOwnProperty(ev.summary)) {
+                calEvents["publicHolidays"][ev.summary] = []
+              }
+
+              for (let i in ev.attendee) {
+                calEvents["publicHolidays"][ev.summary].push(`>*${ev.attendee[i].params.CN}*`);
+              }
             }
           }
         }
       }
 
-      resolve(vacations);
+      resolve(calEvents);
     });
   });
 });
 
 Promise.all(icalPromises).then((values) => {
-  const vacations =  [].concat.apply([], values).sort();
-  console.log(vacations);
-
-  if (vacations.length > 0) {
-    const webhook = new IncomingWebhook(slackWebhook);
-
-    // Send the notification
-    (async () => {
-      await webhook.send({
-        text: vacations.join("\n"),
-      });
-    })();
+  vacations = merge.all(values)
```

Comment:
can you please add `const` before the declaration?

Reply:
Done!